ui/app/send.js

```diff
@@ -239,6 +317,7 @@ SendTransactionScreen.prototype.onSubmit = function () {
   var txParams = {
     from: this.props.address,
     value: '0x' + value.toString(16),
+    gasMultiplier: gasMultiplier * 0.01,
```

Comment:
I just checked, it converts to a number and returns a number.  That's news to me!